CMakeLists.txt

```diff
@@ -600,6 +600,18 @@ else()
   message(STATUS "JNI library is disabled")
 endif()
 
+# Installation and packaging for Linux
+include(GNUInstallDirs)
+install(TARGETS ${ROCKSDB_STATIC_LIB} COMPONENT devel ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
```

Comment:
Installing simultaneously two versions (shared and static) of the same library in my opinion is a popular CMake antipattern. `add_library` + `BUILD_SHARED_LIBS=ON/OFF` should be used instead:
* http://cgold.readthedocs.io/en/latest/tutorials/libraries/static-shared.html